cmd/createcluster.go

```diff
@@ -164,6 +169,10 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 			return err
 		}
 
+		if len(def.DepositAmounts) == 0 {
+			def.DepositAmounts = deposit.EthsToGweis(conf.DepositAmounts)
```

Comment:
we can't modify a definition object that is loaded from a file as it will invalidate the definition hash in the def object

Reply:
I understand this would change the hash, but I see in other places we do this, e.g. https://github.com/ObolNetwork/charon/blob/3e12d09b357b1679788a517ba29836f05eff670b/cmd/createcluster.go#L215
Anyway for extra safety I reworked this.